extensions/big-data-cluster/src/bigDataCluster/utils.ts

```diff
@@ -71,36 +71,50 @@ export function showErrorMessage(error: any, prefixText?: string): void {
 	}
 }
 
+/**
+ * Mappings of the different expected state values to their localized friendly names.
+ * These are defined in aris/projects/controller/src/Microsoft.SqlServer.Controller/StateMachines
+ */
+const stateToDisplayTextMap = {
+	// K8sScaledSetStateMachine
+	'creating': localize('state.creating', "Creating"),
+	'waiting': localize('state.waiting', "Waiting"),
+	'ready': localize('state.ready', "Ready"),
+	'deleting': localize('state.deleting', "Deleting"),
+	'deleted': localize('state.deleted', "Deleted"),
+	'applyingupgrade': localize('state.applyingUpgrade', "Applying Upgrade"),
+	'upgrading': localize('state.upgrading', "Upgrading"),
+	'applyingmanagedupgrade': localize('state.applyingmanagedupgrade', "Applying Managed Upgrade"),
+	'managedupgrading': localize('state.managedUpgrading', "Managed Upgrading"),
+	'rollback': localize('state.rollback', "Rollback"),
+	'rollbackinprogress': localize('state.rollbackInProgress', "Rollback In Progress"),
+	'rollbackcomplete': localize('state.rollbackComplete', "Rollback Complete"),
+	'error': localize('state.error', "Error"),
+
+	// BigDataClusterStateMachine
+	'creatingsecrets': localize('state.creatingSecrets', "Creating Secrets"),
+	'waitingforsecrets': localize('state.waitingForSecrets', "Waiting Fo Secrets"),
```

Comment:
Typo `Waiting For Secrets`

Reply:
Here I was just trying to be hip and cool with the lingo too